app/controllers/grant_submissions/submissions_controller.rb

```diff
@@ -38,20 +38,15 @@ def edit
     end
 
     def create
-      if @grant.form.disabled
```

Comment:
are you removing the ability the disable a form because there's only ever going to be one form for a grant?